client/src/pages/profile/index.tsx

```diff
@@ -0,0 +1,494 @@
+                <Button
+                  size="medium"
+                  className="edit-profile-btn"
+                  onClick={startEditMode}
+                  content="Edit Profile"
+                />
+              )}
+
+              <Image
+                size="small"
+                circular
+                src={
+                  user.profilePic !== '' && user.profilePic !== null
+                    ? user.profilePic
+                    : DefaultProfile
+                }
+              />
+              {user.masthead && (
+                <Image
+                  style={{ marginTop: '-25px' }}
+                  size="small"
+                  src={Masthead}
+                />
+              )}
+              {isEditMode && (
+                <Button.Group>
+                  <Button
+                    className="edit-mode-btn save"
+                    content="Save Changes"
+                    onClick={updateUser}
+                    size="medium"
+                  />
+                  <Button
+                    className="edit-mode-btn cancel"
+                    content="Cancel Changes"
+                    onClick={cancelEditMode}
+                    size="medium"
+                  />
+                </Button.Group>
+              )}
+            </Grid.Column>
+            <Grid.Column width={5}>
+              <HeaderTag as="h2">Basic Information</HeaderTag>
+              <Container style={{ marginBottom: '5px' }}>
+                <div className="input-field">
+                  <span>{`${parseCamelCase('firstName')}:`}</span>
+                  <Input
+                    value={user.firstName}
+                    transparent
+                    readOnly={!isEditMode}
+                    onChange={(e, { value }) =>
+                      setUser(updateUserField(user, 'firstName', value))
+                    }
+                  />
+                </div>
+                <div className="input-field">
+                  <span>{`${parseCamelCase('lastName')}:`}</span>
+                  <Input
+                    value={user.lastName}
+                    transparent
+                    readOnly={!isEditMode}
+                    onChange={(e, { value }) =>
+                      setUser(updateUserField(user, 'lastName', value))
+                    }
+                  />
+                </div>
+                <div className="input-field">
+                  <span>{`${parseCamelCase('preferredName')}:`}</span>
+                  <Input
+                    value={user.preferredName}
+                    transparent
+                    readOnly={!isEditMode}
+                    onChange={(e, { value }) =>
+                      setUser(updateUserField(user, 'preferredName', value))
+                    }
+                  />
+                </div>
+                {isEditMode ? (
+                  <div className="input-field">
+                    <span>{parseCamelCase('genders')}</span>
```

Comment:
What's the value in doing it this way over just typing it how it's supposed to appear?

Reply:
Good catch, none actually. But when I implemented it the first time, I was looping through the keys I needed to print and then using the parseCamelCase to convert the value I was on to the label generically. I'd like to get this back to a generalized loop again but I need some other PRs to be merged first. Specifically, the admin-only profile fields